src/test/kotlin/lotto/LottoTest.kt

```diff
@@ -0,0 +1,50 @@
+package lotto
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class LottoTest {
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-lotto/pull/1#discussion_r463593950

```suggestion
class LottoGameTest {
```